chapters/VK_EXT_present_timing/queries.adoc

```diff
@@ -0,0 +1,481 @@
+    If the pname:swapchain is using relative present times, it is zero and
+    the actual present duration can: be obtained by comparing the
+    pname:pPresentStages results of consecutive presents.
+  * pname:presentStageCount is a count of items contained in
+    pname:pPresentStages.
+  * pname:pPresentStages is `NULL` or a pointer to an array of
+    slink:VkPresentStageTimeEXT providing timing information for each of the
+    present stages requested in
+    slink:VkSwapchainPresentTimingCreateInfoEXT::pname:presentStageQueries.
+  * pname:timingPropertiesChanged is ename:VK_TRUE if the swapchain's
+    timing properties have changed since the last time those
+    properties were queried with
+    flink:vkGetSwapchainTimingPropertiesEXT, otherwise the properties
+    have not changed.
+  * pname:timeDomainChanged is ename:VK_TRUE if the time domain enabled for
+    the swapchain is not currently available.
+  * pname:reportComplete is ename:VK_TRUE if the presentation engine
+    has reported all the requested results in pname:pPresentStages.
+
+Timing information for some present stages may: have a time value of 0,
+indicating that results for that present stage are not available.
+
+For systems with multiple entities operating downstream of the presentation
+engine, such as multiple displays, pname:pPresentStages will return timing
+results for at least one entity which has been affected by the presentation.
+
+If pname:pPresentStages is `NULL`, then the number of available present
+stages reports is returned in pname:presentStageCount. Otherwise,
+pname:presentStageCount must: be set by the application to the number of
+elements in the pname:pPresentStages array, and on return the variable is
+overwritten with the number of entries actually written to
+pname:pPresentStages. If the value of pname:presentStageCount is less than
+the number of available present stages reports, at most
+pname:presentStageCount will be written to pname:pPresentStages.
+
+If pname:timeDomainChanged is ename:VK_TRUE, the application must: query
+what time domains are available and enable present timing with a
+currently-available time domain. If the currently-enabled time domain is the
+opaque domain of ename:VK_TIME_DOMAIN_SWAPCHAIN_LOCAL_EXT, it is possible
+that ename:VK_TIME_DOMAIN_SWAPCHAIN_LOCAL_EXT will be returned by
+flink:vkGetSwapchainTimeDomainsEXT. In such a case, the presentation engine
+may: have multiple opaque time domains that it is switching between.
+
+If any of pname:timeDomainChanged or pname:timingPropertiesChanged is
+ename:VK_TRUE, an application must: not compare the values of
+pname:pPresentStages with any other values, as the presentation engine may
+not be able to provide accurate values.
+
+The results for a given pname:swapchain and present id are returned from
+fname:vkGetPastPresentationTimingEXT until pname:reportComplete is
+ename:VK_TRUE and all available results have been reported in
+pname:pPresentStages.
```

Comment:
Question (that I don't think applies to Android): if timing data for 3 stages is asked for, and if that data trickles in across 3 calls to vkGetPastPresentationTimingEXT(), will the final call to vkGetPastPresentationTimingEXT() contain the data for all 3 stages, or will the application need to cache and combine that info?